core/src/main/java/bisq/core/payment/AustraliaPayid.java

```diff
@@ -17,53 +17,37 @@
 
 package bisq.core.payment;
 
-import bisq.core.payment.payload.AustraliaPayIDPayload;
+import bisq.core.locale.FiatCurrency;
+import bisq.core.payment.payload.AustraliaPayidPayload;
 import bisq.core.payment.payload.PaymentAccountPayload;
 import bisq.core.payment.payload.PaymentMethod;
-import bisq.core.payment.payload.AustraliaPayIDPayload;
-
-import org.jetbrains.annotations.NotNull;
 
-import lombok.Getter;
-import lombok.Setter;
-
-import bisq.core.locale.Country;
-import bisq.core.locale.FiatCurrency;
-import bisq.core.payment.payload.AustraliaPayIDPayload;
-
-public final class AustraliaPayID extends PaymentAccount
-{
-    public AustraliaPayID()
-    {
+public final class AustraliaPayid extends PaymentAccount {
+    public AustraliaPayid() {
         super(PaymentMethod.AUSTRALIA_PAYID);
         setSingleTradeCurrency(new FiatCurrency("AUD"));
     }
 
     @Override
-    protected PaymentAccountPayload createPayload()
-    {
-        return new AustraliaPayIDPayload(paymentMethod.getId(), id);
+    protected PaymentAccountPayload createPayload() {
+        return new AustraliaPayidPayload(paymentMethod.getId(), id);
     }
 
-    // payid
-    public String getPayID()
-    {
-        return ((AustraliaPayIDPayload) paymentAccountPayload).getPayid();
+    public String getPayid() {
+        return ((AustraliaPayidPayload) paymentAccountPayload).getPayid();
     }
-    public void setPayID(String payid)
-    {
+
+    public void setPayid(String payid) {
```

Comment:
Our naming convention would be: `setPayId`  and `payId`
Would require also changes in the protobuf

Reply:
I think payid is one word in this case. It's not an id as part of bisq, just the name of the payment method.

Reply:
True. They use PayID... anyway not sure whats best here, but also not really important...

Reply:
I had it like that before, but I agree with @sqrrm that it is actually a brand name and should be without camelcase in that case.